rpc/api/debug_js.go

```diff
@@ -25,37 +25,37 @@ web3._extend({
 			name: 'printBlock',
 			call: 'debug_printBlock',
 			params: 1,
-			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
+			inputFormatter: [null]
```

Comment:
The question really is whether this breaks the old API or not? I modified it because it wouldn't play nice with the new methods, but I don't know about the old ones.